test/Intl/CollatorUnsupportedLocales.js

```diff
@@ -3,35 +3,28 @@
 // Licensed under the MIT license. See LICENSE.txt file in the project root for full license information.
 //-------------------------------------------------------------------------------------------------------
 
-var passed = true;
+WScript.LoadScriptFile("..\\UnitTestFramework\\UnitTestFramework.js");

```

Comment:
Appreciate the change to use the UnitTestFramework -- but just checking, was this test broken before?

Reply:
Sortof -- note that it used to test best fit twice, rather than best fit followed by lookup (which is what the assert messages suggested it should do). Since its the same code both times and it was obviously a copy-paste bug, I thought itd be best to factor it out regardless.